packages/application-tester/src/provider-test-helper.ts

```diff
@@ -0,0 +1,22 @@
+export interface ApplicationOutputs {
+  graphqlURL: string
+  websocketURL: string
+}
+
+export interface Counters {
+  subscriptions(): Promise<number>
+  connections(): Promise<number>
+  readModels(readModelName: string): Promise<number>
+  events(): Promise<number>
+}
+
+export interface Queries {
+  events(primaryKey: string): Promise<Array<unknown>>
+  readModels(primaryKey: string, readModelName: string): Promise<Array<unknown>>
+}
+
+export interface ProviderTestHelper {
+  outputs: ApplicationOutputs
+  counters: Counters
+  queries: Queries
+}
```

Comment:
These are the interfaces each provider needs to implement so that they can use the same integration tests we already have